content/en/tracing/guide/send_traces_to_agent_by_api.md

```diff
@@ -58,7 +58,7 @@ and each span is a dictionary with a `trace_id`, `span_id`, `resource` and so on
 | `service`    | string  | The service you are tracing. The service name must not be longer than 100 characters. |
 | `span_id`    | int64   | The span integer (64-bit unsigned) ID. |
 | `start`      | int64   | The start time of the request in nanoseconds from the UNIX epoch. |
-| `trace_id`   | int64 or int128   | The unique integer (64-bit unsigned or 128-bit unsigned) ID of the trace containing this span. |
+| `trace_id`   | int64   | The lower 64-bits of the unique integer ID of the trace containing this span. Set any upper 64-bits by setting the tag `_dd.p.tid` in hex-encoded lowercase in `meta`. |
```

Comment:
Not clear to me whether in `_dd.p.tid` the user should set only the upper 64 bits or all 128 bits hex-encoded.